src/unix/bsd/apple/mod.rs

```diff
@@ -4066,14 +4066,14 @@ pub const AF_E164: c_int = AF_ISDN;
 pub const pseudo_AF_KEY: c_int = 29;
 pub const AF_INET6: c_int = 30;
 pub const AF_NATM: c_int = 31;
-pub const AF_SYSTEM: c_int = 32;
+pub const AF_SYSTEM: c_uchar = 32;
 pub const AF_NETBIOS: c_int = 33;
 pub const AF_PPP: c_int = 34;
 pub const pseudo_AF_HDRCMPLT: c_int = 35;
 pub const AF_IEEE80211: c_int = 37;
 pub const AF_UTUN: c_int = 38;
 pub const AF_VSOCK: c_int = 40;
-pub const AF_SYS_CONTROL: c_int = 2;
+pub const AF_SYS_CONTROL: c_ushort = 2;
```

Comment:
```suggestion
pub const AF_SYS_CONTROL: c_ushort = 2;
```

It looks like the field is `u_int16_t`, so this can just be `u16` https://developer.apple.com/documentation/kernel/sockaddr_ctl/1809252-ss_sysaddr.

Reply:
Updated. Waiting on CI.

Reply:
The CI test failed after I change `AF_SYS_CONTROL` to `u16`. Any idea why? @tgross35

Reply:
It doesn't look related, the android CI has been flaky.